tests/e2e-leg-6/verify-server-logrotate/expected-output/logrotateTimerServiceOut.txt

```diff
@@ -37,11 +37,6 @@ ALTER DATABASE
 (1 row)
 
 ALTER DATABASE
-        hurry_service         
```

Comment:
It's not that it turns off the time service; rather, it triggers the hurry_service to run immediately instead of waiting for the next scheduled run. Additionally, we clean up the configuration, so it shouldn't have any downstream effects.

If we encounter any issues in the future, we can increase the timing instead of removing it, since we have a timeout issue in the latest server image `Service hurried but not finished after waiting for 60 seconds`